big_tests/rebar.config

```diff
@@ -19,6 +19,7 @@
         {shotgun, "0.4.0"},
         {amqp_client, "3.8.0"},
         {esip, "1.0.30"},
+        {ibrowse, "4.4.1"},
```

Comment:
Why do we need `ibrowse`? From what I can see, this was not maintained since Sep 2018.

Reply:
We have `gun` in big_tests (not as a direct dependency of `big_tests`) which, IMO, is quite good and maintained HTTP library.

Reply:
first of all our [fusco](https://github.com/esl/fusco) project is not very active as well, but that doesn't mean that it's not mantained. ibrowse it quite known and used project.

gun and other similar libraries are designed to create long term connection for frequent requests, also we would have to cat out the base address from the returned links. ibrowse is brillian when you need to just need to test just one request. i think it's much better for testing.

Reply:
All right, let it be. We'll change it to sth maintained when or if it stops working with a future version of Erlang.
And I have doubts about `fusco` being maintained, but that's a different story.